ruby_connectors/lib/ruby_connectors/temporal/workflows/fetch_schema_workflow.rb

```
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module RubyConnectors
+  module Temporal
+    module Workflows
+      class FetchSchemaWorkflow < ::Temporal::Workflow
+        def execute(connector_id)
+
+        RubyConnectors::Temporal::Activities::FetchSchemaActivity.execute!(connector_id)
```

rubocop

[convention] Extra empty line detected at method body beginning.

(Layout/EmptyLinesAroundMethodBody)